zebra-script/src/lib.rs

```diff
@@ -38,6 +41,9 @@ pub enum Error {
     /// tx is an invalid size for it's protocol
     #[non_exhaustive]
     TxSizeMismatch,
+    /// script is invalid
+    #[non_exhaustive]
+    TxInvalidScript,
```

Comment:
Naming: Can we use a more specific name, to avoid confusion with `ScriptInvalid`?

```suggestion
    InvalidOutputScriptType,
```

Reply:
Good call, fixed in 935e704ce - I left the `Tx` prefix to be consistent with the rest